blog/EclipseFuseTooling-NewGlobalConfiguration.adoc

```diff
@@ -0,0 +1,30 @@
+

```

Comment:
another one - you no longer need to repeat the date in the file name - just have :page-date in the preamble.